apps/system/utils/Kconfig

```diff
@@ -50,6 +50,31 @@ config NET_PING_CMD_ICOUNT
 
 endif #NET_CMDS
 
+config ENABLE_CPULOAD
+	bool "cpuload"
```

Comment:
ENABLE_CPULOAD is not appropriate for the intended work,
Better to use CPULOAD_MONITOR here..
What is your opinion ?

Reply:
ENABLE_SOMETHING is a kind of naming convention here. It seems good to me.

Reply:
I think @shadaksharikp is right. Bcz we already have CONFIG_SCHED_CPULOAD and it has to be enabled to enabel this config. But this config name conveys that enabel cpul load. It is better change the name of this config ex: ENABLE_CPULOAD_MONITOR or something appropriate

Reply:
As davidfather said, currently all kdbg commands have the config name, ENABLE_XXXX.
At the same time, I agree with confusion with CONFIG_SCHED_CPULOAD.
But I think CPULOAD_MONITOR is not appropriate for that case because this command displays cpu load once as well as executes cpu load monitor daemon.
I think the 'monitor' word feels that it measures cpuload with running continously.  
In conclusion, How about changing from ENABLE_XXXX to ENABLE_XXXX_CMD.
You guys, please give me your opinion :)

Reply:
Hello @jeongarmy,
Main purpose of this kdbg command is to dispaly the CPU load statistics. 
Hence in my opinion, ENABLE_CPUSTAT would be good.

Reply:
@manoharahk Linux has "cpustat". But it is different with ours. And stat is too big word to show our functionality. cpuload shows perfectly what it is. I give @jeongarmy  my vote, ENABLE_XXX_CMD.

Reply:
Hi @manohara, thank you for your help.
The config name of all kdbg commands are related to name used as tash command actually.
 I think it can give some confusion